recipes/reframed/meta.yaml

```diff
@@ -43,7 +45,5 @@ about:
 
 extra:
   recipe-maintainers:
-    - acaprez
```

Comment:
I’m assuming you have a good reason for removing this user from the maintainer list, but just for completeness, can you state it?

Reply:
Hi! I am new here, so I am sorry if something wasn't done according to the good practices.
The new maintainer I added is the developer of reframed and is fine with being added as a maintainer here. 
I don't have a good reason to remove this user, I didn't contact him at all. Now that I think about it, I can see these might not be the appropriate way to go... so I will contact this user.

Reply:
Either way is fine with me.  If the developer of the package wants to directly be the recipe maintainer themselves, I have no objection and you can remove me from the list.

Reply:
I think it is fine. Thanks!